server/src/logic.js

```diff
@@ -59,7 +59,10 @@ export const userHandler = {
   query(_, __, ctx) {
     return getAuthenticatedUser(ctx);
   },
-  groups(user) {
+  groups(user, args) {
+    if (args.id) {
```

Comment:
Let it be known that I don't think this is the right place for this - it belongs as a root-level query but we can remove it later once we have an alternative